src/napari_flim_phasor_calculator/_reader.py

```diff
@@ -78,9 +87,14 @@ def flim_file_reader(path):
             sdt_file = sdtfile.SdtFile(path)  # header to be implemented
             data_raw = np.asarray(sdt_file.data)  # option to choose channel to include
             data = np.moveaxis(np.stack(data_raw), 3, 1)
-            metadata = {name: sdt_file.measure_info[0][name].item() for name in sdt_file.measure_info[0].dtype.names}  # convert recarray to dict
-            metadata['times'] = sdt_file.times
-            metadata['file_type'] = 'sdt'
+
+            # create list of measure_info for each channel
+            measure_info_list = []
+            for measure_info_recarray in sdt_file.measure_info:
```

Comment:
Does this method `sdt_file.measure_info` always returns a list with length 1?
Or does it return a different element if the image has more channels?

Reply:
I tested with the 2 channel image, and indeed it provides more metadata then.